src/test/java/SimpleRetrofitForChecking/RequestPost.java

```diff
@@ -0,0 +1,26 @@
+package SimpleRetrofitForChecking;
```

Comment:
please add comments on what was done as part of the SimpleRetrofitForChecking package